zokrates_parser/src/zokrates.pest

```diff
@@ -1,5 +1,5 @@
 
-file = { SOI ~ NEWLINE* ~ pragma? ~ NEWLINE* ~ import_directive* ~ NEWLINE* ~ ty_struct_definition* ~ NEWLINE* ~ function_definition* ~ EOI }
+file = { SOI ~ NEWLINE* ~ pragma? ~ NEWLINE* ~ import_directive* ~ NEWLINE* ~ ty_struct_definition* ~ NEWLINE* ~ const_definition* ~ NEWLINE* ~ function_definition* ~ EOI }
```

Comment:
Let's take some time to think of the implications of this ordering (imports > structs > constants > functions) and see if we can relax some of it. Maybe in a separate issue.

Reply:
Yes, I was thinking about changing this, we should discuss this further

Reply:
Let's do it in another one, this one is quite big already